manual/source/conf.py

```diff
@@ -46,6 +46,7 @@
     'sphinx.ext.ifconfig',
     'sphinx.ext.viewcode',
     'sphinx.ext.githubpages',
+    'nbsphinx'
```

Comment:
Is this the [*heavy lifter*](https://nbsphinx.readthedocs.io/en/0.8.8/) that brings notebooks into the documentation automagically (without need for a `Makefile`)?

Reply:
Indeed, the sphinx extension `nbsphinx` basically turns all referenced jupyter notebooks in source files for documentation.